src/tests/components/FieldGroup.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import { Provider } from 'react-redux'
+import FieldGroup from '../../components/FieldGroup'
+import ProjectForm from '../../components/ProjectForm'
+import store from '../../store'
+
+describe('FieldGroup', () => {
+  const testFields = [
+    {
+      name: 'Test 1'
+    },
+    {
+      name: 'Test 2'
+    }
+  ]
+  const meta = { error: {}, submitFailed: false }
+  const type = 'testType'
+  const start = 'testStart'
+
+  it('renders the component with no errors', () => {
+    const wrapper = shallow(
+      <FieldGroup fields={testFields} meta={meta} />
+    )
+    expect(wrapper.find('Field').length).toEqual(2)
+  })
+
+  it('renders the first field with the correct label', () => {
+    const wrapper = shallow(
+      <FieldGroup fields={testFields} meta={meta} type={type} start={start} />
+    )
+    expect(wrapper.find('Field').filterWhere(field => {
+      return field.prop('label') === start + ' (primary)'
+    }).length).toEqual(1)
+  })
+
+  it('renders the second field with the correct label', () => {
+    const wrapper = shallow(
+      <FieldGroup fields={testFields} meta={meta} type={type} start={start} />
+    )
+    expect(wrapper.find('Field').filterWhere(field => {
+      return field.prop('label') === start + ' (2)'
+    }).length).toEqual(1)
+  })
+
+  it('renders the first field with the correct name', () => {
+    const wrapper = shallow(
+      <FieldGroup fields={testFields} meta={meta} type={type} start={start} />
+    )
+    expect(wrapper.find('Field').filterWhere(field => {
+      return field.prop('name') === `${type}s[0].value`
+    }).length).toEqual(1)
+  })
+
+  it('renders the second field with the correct label', () => {
+    const wrapper = shallow(
+      <FieldGroup fields={testFields} meta={meta} type={type} start={start} />
+    )
+    expect(wrapper.find('Field').filterWhere(field => {
+      return field.prop('label') === start + ' (2)'
+    }).length).toEqual(1)
+  })
+
+  it('adds a field', () => {
```

Comment:
does it make sense to say `supports field addition` or something similar that portrays the idea that the user triggers the action?